docs/appendices/migration-guides/index.mdx

```diff
@@ -27,6 +27,21 @@ Please check if you have overriden the `AddressForm` component
 ([commit](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/fd27e93d3e345157251ac6d3ace5e5e56c528ce6))
 if you're interested in using this feature.
 
+### Node 18 support
+
+Front-Commerce officially supports Node 18 starting from version 2.20. However,
+as with Node 17, it still requires you to prefix your npm commands by
+`NODE_OPTIONS=--openssl-legacy-provider` e.g.
+`NODE_OPTIONS=--openssl-legacy-provider npm start`,
+`NODE_OPTIONS=--openssl-legacy-provider npm run build`.
+
+You can also put it directly in your `.npmrc` file at the root of your
+repository:
+
+```title=".npmrc"
+node-options=--openssl-legacy-provider
+```
```

Comment:
This can be removed once https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/1743 is merged.